lib/mongoid/locker.rb

```diff
@@ -287,8 +295,8 @@ def lock!(opts)
       end
     end
 
-    def unlock!
-      Mongoid::Locker::Wrapper.find_and_unlock(self)
+    def unlock!(opts)
```

Comment:
Generally default opts, `opts = {}`, especially if this is an external interface change.

Reply:
`unlock!` is protected method, and it passes `opts` into `find_and_unlock` where `opts` parameter is mandatory.